packages/react-components/react-message-bar/library/src/components/MessageBarBody/MessageBarBody.types.ts

```diff
@@ -12,4 +18,4 @@ export type MessageBarBodyProps = ComponentProps<MessageBarBodySlots>;
 /**
  * State used in rendering MessageBarBody
  */
-export type MessageBarBodyState = ComponentState<MessageBarBodySlots>;
+export type MessageBarBodyState = ComponentState<MessageBarBodySlots> & { linkInline?: boolean };
```

Comment:
```suggestion
export type MessageBarBodyState = ComponentState<MessageBarBodySlots>;
```

Let's avoid exposing on `state` sooner than it's requested :)